src/routes/safe/components/Settings/ManageOwners/AddOwnerModal/index.tsx

```diff
@@ -14,19 +13,32 @@ import { TX_NOTIFICATION_TYPES } from 'src/logic/safe/transactions'
 import addSafeOwner from 'src/logic/safe/store/actions/addSafeOwner'
 import createTransaction from 'src/logic/safe/store/actions/createTransaction'
 
-import { safeOwnersSelector, safeParamAddressFromStateSelector } from 'src/logic/safe/store/selectors'
+import { safeParamAddressFromStateSelector } from 'src/logic/safe/store/selectors'
 import { checksumAddress } from 'src/utils/checksumAddress'
 import { makeAddressBookEntry } from 'src/logic/addressBook/model/addressBook'
+import { Dispatch } from 'redux'
```

Comment:
I think we should be using this here
https://github.com/gnosis/safe-react/blob/development/src/logic/safe/store/actions/types.d.ts

Reply:
Thanks! it's done